microsoft.ui.xaml/uielement_keyboardaccelerators.md

```diff
@@ -20,15 +20,15 @@ Accelerators are typically assigned to buttons or menu items.
 
 ## -property-value
 
-The collection of [**KeyboardAccelerator**](../microsoft.ui.xaml.input/keyboardaccelerator.md) objects.
+The collection of **[KeyboardAccelerator]**(../microsoft.ui.xaml.input/keyboardaccelerator.md) objects.
```

Comment:
This is inconsistent in the docs, but I believe the style guide recommends not using bold with links.
```suggestion
The collection of [KeyboardAccelerator](../microsoft.ui.xaml.input/keyboardaccelerator.md) objects.
```